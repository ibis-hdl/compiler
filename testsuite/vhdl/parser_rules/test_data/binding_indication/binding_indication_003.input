-- optional entity aspect
USE ENTITY foo

