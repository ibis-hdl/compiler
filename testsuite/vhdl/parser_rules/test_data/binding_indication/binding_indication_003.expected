(binding_indication
  (entity_aspect<v>
    (entity_aspect::entity
      (name<v>
        (identifier
          foo
        )
      )
    )
  )
  (generic_map_aspect
    (association_list
      ()
    )
  )
  (port_map_aspect
    (association_list
      ()
    )
  )
)
